packages/rollup-plugin/test/__snapshots__/rollup-plugin.test.ts.snap

```diff
@@ -554,6 +554,303 @@ export { altTheme, responsiveTheme, theme, vars };
+      text: "papayawhip"
+    },
+    space: {
+      1: "5px",
+      2: "10px",
+      3: "15px"
+    }
+  });
+  setElementVars(dynamicVarsContainer, {
+    [vars.colors.backgroundColor]: "darksalmon"
+  });
+}
+render();
+",
+  ],
+  [
+    "src/shared.css.js",
+    "import { setFileScope, endFileScope } from '@vanilla-extract/css/fileScope';
+import { style, globalStyle } from '@vanilla-extract/css';
+
+setFileScope("src/shared.css.ts", "@fixtures/themed");
+const shadow = style({
+  boxShadow: "0 0 5px red"
+}, "shadow");
+globalStyle("body", {
+  backgroundColor: "skyblue"
+});
+globalStyle("body, button", {
+  lineHeight: "16px"
+});
+endFileScope();
+
+export { shadow };
+",
+  ],
+  [
+    "src/styles.css.js",
+    "import { setFileScope, endFileScope } from '@vanilla-extract/css/fileScope';
+import { fontFace, globalFontFace, style, fallbackVar, globalStyle, createVar, styleVariants } from '@vanilla-extract/css';
+import { shadow } from './shared.css.js';
+import { vars, altTheme, theme } from './themes.css.js';
+
+setFileScope("src/styles.css.ts", "@fixtures/themed");
+const impact = fontFace({
+  src: 'local("Impact")'
+}, "impact");
+globalFontFace("MyGlobalComicSans", {
+  src: 'local("Comic Sans MS")'
+});
+const container = style({
+  display: "flex",
+  flexDirection: "column",
+  gap: vars.space[2],
+  padding: vars.space[3],
+  "@media": {
+    "only screen and (min-width: 500px)": {
+      border: \`1px solid \${vars.colors.backgroundColor}\`
+    }
+  }
+}, "container");
+const iDunno = style([{
+  zIndex: 1
+}, {
+  position: "relative"
+}], "iDunno");
+const button = style([{
+  fontFamily: impact,
+  backgroundColor: fallbackVar(vars.colors.backgroundColor, '"THIS FALLBACK VALUE SHOULD NEVER BE USED"'),
+  color: vars.colors.text,
+  borderRadius: "9999px",
+  "@media": {
+    "only screen and (min-width: 500px)": {
+      padding: vars.space[1]
+    },
+    "only screen and (min-width: 1000px)": {
+      padding: vars.space[2]
+    }
+  },
+  selectors: {
+    [\`\${altTheme} \${theme} \${container} &\`]: {
+      fontFamily: "MyGlobalComicSans",
+      outline: "5px solid red"
+    }
+  }
+}, shadow, iDunno], "button");
+globalStyle(\`body \${iDunno}:after\`, {
+  content: "'I am content'"
+});
+const blankVar1 = createVar({
+  syntax: "<number>",
+  inherits: false,
+  initialValue: "0.5"
+}, "blankVar1");
+const blankVar2 = createVar("blankVar2");
+const opacity = styleVariants({
+  "1/2": blankVar1,
+  "1/4": fallbackVar(blankVar1, blankVar2, "0.25")
+}, (value) => ({
+  selectors: {
+    "html &": {
+      opacity: value
+    }
+  }
+}), "opacity");
+endFileScope();
+
+export { button, container, opacity };
+",
+  ],
+  [
+    "src/themes.css.js",
+    "import { setFileScope, endFileScope } from '@vanilla-extract/css/fileScope';
+import { style, createGlobalTheme, createTheme, layer, assignVars } from '@vanilla-extract/css';
+
+setFileScope("src/themes.css.ts", "@fixtures/themed");
```

Comment:
This is the main effect of the filescope injection (as well as the `endFileScope` call at the end of the file).